src/scripts/main.js

```diff
@@ -1 +1,71 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+const firstPromise = new Promise((resolve, reject) => {
+  body.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(reject, 3000, 'First promise was rejected');
+});
+
+const secondPromise = new Promise((resolve) => {
+  body.addEventListener('click', () => {
+    resolve('Second promise was resolved');
+  });
+
+  body.addEventListener('contextmenu', (e) => {
+    e.preventDefault();
+    resolve('Second promise was resolved');
+  });
+});
+
+const rightClickPromise = new Promise(resolve => {
```

Comment:
You don't need to create additional Promises. You need to create just 3 of them.